templates/default/parallelcluster_supervisord.conf.erb

```diff
@@ -32,6 +34,8 @@ environment = HOME="<%= node['cluster']['dcv']['authenticator']['user_home'] %>"
 <% if node['cluster']['scheduler'] == 'slurm' -%>
 [program:computemgtd]
 command = <%= node['cluster']['node_virtualenv_path'] %>/bin/computemgtd
+user = <%= node['cluster']['cluster_admin_user'] %>
+environment = HOME="/home/<%= node['cluster']['cluster_admin_user'] %>",USER="<%= node['cluster']['cluster_admin_user'] %>"
```

Comment:
Supervisord does not change envars when running a command as a specific user, see [documentation](http://supervisord.org/subprocess.html#subprocess-environment).

I've added the very basic ones for the following reasons:
1. it would be incorrect and error prone to run a command as pcluster, but having HOEM and USER set as it was root.
2. it will be easier for a dev to troubleshoot/fix possible envars related issues, having such a line in the supervisord config file.